src/gui/src/KeySequence.cpp

```diff
@@ -211,23 +213,22 @@ QString KeySequence::keyToString(int key)
     // treat key pad like normal keys (FIXME: we should have another lookup table for keypad keys instead)
      key &= ~Qt::KeypadModifier;
 
+    // a special key?
+    int i = 0;
+    while (keyname[i].name) {
+        if (key == keyname[i].key)
+            return QString::fromUtf8(keyname[i].name);
+        i++;
+    }
+
     // a printable 7 bit character?
-     if (key < 0x80 && key != Qt::Key_Space)
+     if (key < 0x80)
```

Comment:
hi, pr description has no explanation on this line change, please could you add it?

Reply:
I edited the pr description to include the explanation